tools/ci_build/github/azure-pipelines/stages/nuget-linux-cuda-packaging-stage.yml

```diff
@@ -70,15 +70,81 @@ stages:
         condition: 'succeeded'
     - template: ../templates/clean-agent-build-directory-step.yml
   # Linux CUDA with TensorRT Packaging
-  - template: jobs/linux-gpu-tensorrt-packaging-job.yml
-    parameters:
-      artifactName: 'onnxruntime-linux-x64-tensorrt-$(OnnxRuntimeVersion)'
-      artifactNameNoVersionString: 'onnxruntime-linux-x64-tensorrt'
-      buildJava: ${{ parameters.buildJava }}
-      buildJavaOption: '--build_java'
-      buildNodejs: ${{ parameters.buildNodejs }}
-      buildNodejsOption: '--build_nodejs'
-      CudaVersion: ${{ parameters.CudaVersion }}
+  - job: Linux_C_API_Packaging_TensorRT
+    dependsOn: []
+    workspace:
+      clean: all
+    timeoutInMinutes: 180
+    pool: 'onnxruntime-Ubuntu2204-AMD-CPU'
```

Comment:
GPU instance not required when building this task?

Reply:
No, gpu is not needed when building, but it is required when testing it.